### PR TITLE
CSIC-32-Multiple Infinibox support

### PR DESCRIPTION
**Pull request by nikhilbarge**
PR for
Multiple Infinibox Support
Updated sideCar versions
added makefile
added error handling for rest calls
other project configurations

### CODE REVIEW

File: deploy/helm/infinibox-csi-driver/templates/controllerserver.yaml [hunk 1]
```diff
@@ -122,11 +125,22 @@ spec:
...
+          env:
+            - name: ADDRESS
+              value: /var/run/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/run/csi      
         - name: driver
-          image: niteshgadekar/driver:1.0
-         # image: {{ required "Provide the Infinibox csi driver container image." .Values.images.csidriver }}
+          image: nikhilbarge/infinibox-csi-driver:test1
```
Comment (Nilesh20) [inline deploy/helm/infinibox-csi-driver/templates/controllerserver.yaml]: Change it to infinidat docker image

File: deploy/helm/infinibox-csi-driver/templates/nodeserver.yaml [hunk 2]
```diff
@@ -78,7 +78,7 @@ spec:
             capabilities:
               add: ["SYS_MODULE"]
             allowPrivilegeEscalation: true
-          image: niteshgadekar/driver:1.0 
+          image: nikhilbarge/infinibox-csi-driver:test1
```
Comment (Nilesh20) [inline deploy/helm/infinibox-csi-driver/templates/nodeserver.yaml]: Image name ?

**Review by Nilesh20 — CHANGES_REQUESTED** [review]
There are lots of file being changed.
Please raise multiplr PR's for such checkin. Run the go fmt for formatting.

**Review by gadekarnitesh — COMMENTED** [review]
lg

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Folders
```
Comment (Nilesh20) [inline .gitignore:1]: This PR has the other changes as well , Please add it's details into message

**Review by Nilesh20 — CHANGES_REQUESTED** [review]

**Review by Nilesh20 — APPROVED** [review]
Approving for now. Please consider raised points going forward.

**Pull request merged**